code/modules/library/lib_items.dm

```diff
@@ -283,6 +284,68 @@
 	else
 		return ..()
 
+/obj/item/weapon/book/mindbook
+	name = "A Harmless Book"
+	icon = 'icons/obj/library.dmi'
+	icon_state ="book"
+	unique = 1
+	forbidden = 1
+	var/hallucinatory_cooldown
+
+/obj/item/weapon/book/mindbook/attack_self(mob/living/H)
+	if(ishuman(H))
+		var/mob/living/carbon/human/P = H
+		if((P.mind && P.mind.assigned_role == "Psychiatrist") || prob(50))
+			if(hallucinatory_cooldown > world.time)
+				to_chat(P, "The book needs to recharge.")
+				return
+			else
+				var/list/victim_list = list()
+				for(var/mob/living/carbon/human/V in view(P.client.view, P))
+					if(!isliving(V))

```

Comment:
This was actually supposed to clear out any dead people, but since I'm dumb, I made it like this. Fixin'